CHANGELOG.md

```diff
@@ -0,0 +1,334 @@
+- Make `SIZE` function work with s-expressions. [#379](https://github.com/partiql/partiql-lang-kotlin/pull/379)
+- A number of other minor bug fixes and technical debt has been addressed. For a complete list of PRs that made it into 
+  this release, please see the v0.3.0 GitHub milestone.
+
+## [0.1.6-alpha] - 2021-05-13
+
+### Fixed
+- Adds Compile-Time Thread.interrupted() checks [#398](https://github.com/partiql/partiql-lang-kotlin/pull/398)
+
+## [0.1.5-alpha] - 2021-04-27
+
+### Fixed
+- Fixes a severe performance issue relating the sanity checks performed on very large queries before compilation. [#391](https://github.com/partiql/partiql-lang-kotlin/pull/391)
+
+## [0.2.6-alpha] - 2021-02-18
+
+### Added
+- Functions to convert from UNIX epoch to TIMESTAMP and TIMESTAMP to UNIX epoch. [#330](https://github.com/partiql/partiql-lang-kotlin/pull/330)
+- Adds a Rewriter to VisitorTransform [guide](https://github.com/partiql/partiql-lang-kotlin/blob/feb84730c64a2ad0f12c57bef3b1c45e21279538/docs/dev/RewriterToVisitorTransformGuide.md)
+
+### Changed
+- Migrates existing `AstRewriters` to PIG’s `VisitorTransform`. [#356](https://github.com/partiql/partiql-lang-kotlin/pull/356)
+
+### Deprecated
+- Deprecates AstRewriter, AstRewriterBase, MetaStrippingRewriter, RewriterTestBase
+
+## [0.2.5-alpha] - 2021-01-12
+
+### Added
+- System stored procedure calls (`EXEC`) [#345](https://github.com/partiql/partiql-lang-kotlin/pull/345). 
+  More details on usage can be found [here](https://github.com/partiql/partiql-spec/issues/17)
+- CLI: version number and commit hash in REPL [#339](https://github.com/partiql/partiql-lang-kotlin/pull/339)
+- CLI: `PARTIQL_PRETTY` output-format for non-interactive use [#349](https://github.com/partiql/partiql-lang-kotlin/pull/349)
+- Document thread safety of `CompilerPipeline` [#334](https://github.com/partiql/partiql-lang-kotlin/pull/334)
+
+### Fixed
+- Parsing of `TRIM` specification keywords (`BOTH`, `LEADING`, and `TRAILING`) [#326](https://github.com/partiql/partiql-lang-kotlin/pull/326)
+- Build failure of `TimestampTemporalAccessorTests` when given a negative year [#346](https://github.com/partiql/partiql-lang-kotlin/pull/346)
+- Running of parameterized tests and other test targets [#338](https://github.com/partiql/partiql-lang-kotlin/pull/338) and [#351](https://github.com/partiql/partiql-lang-kotlin/pull/351)
+
+## [0.2.4-alpha] - 2020-11-08
+
+### Fixed
+- [Fix `LIMIT` clause execution order][#300](https://github.com/partiql/partiql-lang-kotlin/pull/300)
+- [Stop treating date parts as if they are string literals][#317](https://github.com/partiql/partiql-lang-kotlin/pull/317)
+
+## [0.2.3-alpha] - 2020-10-09
+
+### Added
+- `LET` (fom `FROM` clauses) implementation.
+
+### Fixed
+- fix: bigDecimalOf no-ops when given an Ion decimal [#293](https://github.com/partiql/partiql-lang-kotlin/pull/293)
+
+## [0.1.4-alpha] - 2020-09-30
```

Comment:
Link not created by GitHub. https://github.com/partiql/partiql-lang-kotlin/compare/v0.1.3-alpha...v0.1.4-alpha